### ISSUE TITLE
119 mic: number of forwarding state confusion

### ISSUE THREAD

**Issue by eqvinox** [furry13/6man-p-flag#41]
Jared Mauch: appeared confused how much forwarding state this can create on the router. It creates exactly as much as the network hands out by IA_PD. (Might belong in the other draft?)

**Comment by furry13** [comment]
I do not think we need to say anything - the benefits for the network are clearly stated in the -pd v6ops draft.
